KuduSync.NET/KuduSyncOptions.cs

```diff
@@ -39,15 +39,18 @@ public class KuduSyncOptions : CommandLineOptionsBase
         [Option("q", "quiet", Required = false, HelpText = "No logging")]
         public bool Quiet { get; set; }
 
-        [Option("v", "verbose", Required = false, HelpText = "Verbose logging with maximum number of output lines")]
-        public int? Verbose { get; set; }
+        [Option("v", "verbose",  DefaultValue = false, Required = false, HelpText = "Verbose logging with maximum number of output lines")]
+        public bool Verbose { get; set; }
```

Comment:
That looks like a breaking change, since it used to represent the number of lines. Might be best to handle logging changes in a separate PR and keep this one focused on hash